spec/controllers/transactions_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe TransactionsController, type: :controller do
+  describe 'POST create new transaction' do
+    login_user
+    let(:transaction_params) { FactoryBot.attributes_for :transaction }
+
+    it 'has a 200 status code after create' do
+      post :create, params: {transaction: transaction_params}
+      assert_redirected_to root_path
+    end
+  end
+end
```

Comment:
@sinpai 
* we have two actions in the controller: `new` and `create`. You've tested only `create`
* `create` action can 1) render `new` 2) redirect to. So we have to test both cases: creation passed and with errors

Reply:
There is a cool thing for testing stuff that goes before:
```ruby
before do
  ...
end
```
such things as `login_user` should come in such blocks

check [best practisies](http://www.betterspecs.org/)

Reply:
My implementation of `login_user` function is 
```
def login_user
    before(:each) do
      @request.env['devise.mapping'] = Devise.mappings[:user]
      user = FactoryBot.create(:user)
      sign_in user
    end
  end
```
So i need to just call it without `before do; end` block. Is it ok?

Reply:
@sinpai yeah, it's ok 👍